namespace cpp facebook.fboss.thrift
namespace cpp2 facebook.fboss.thrift
namespace d neteng.fboss
namespace go neteng.fboss
namespace php fboss
namespace py neteng.fboss
namespace py.asyncio neteng.asyncio.fboss

exception FbossBaseError {
  1: string message
} ( message = "message" )
